azurerm/provider.go

```diff
@@ -334,6 +336,7 @@ func determineAzureResourceProvidersToRegister(providerList []resources.Provider
 		"Microsoft.DBforPostgreSQL":     {},
 		"Microsoft.Devices":             {},
 		"Microsoft.DocumentDB":          {},
+		"Microsoft.DataLakeStore":       {},
```

Comment:
minor: alphabetically this should be higher up

Reply:
moved in next commit